mlir/test/lib/Dialect/Vector/TestVectorTransforms.cpp

```diff
@@ -489,7 +489,9 @@ struct TestFlattenVectorTransferPatterns
   Option<unsigned> targetVectorBitwidth{
       *this, "target-vector-bitwidth",
       llvm::cl::desc(
-          "Minimum vector bitwidth to enable the flattening transformation"),
+          "Minimum vector bitwidth to enable the flattening transformation. "
+          "For scalable vectors this is the base size that's known at compile "
+          "time."),
```

Comment:
```suggestion
          "For scalable vectors this is the base size (vscale=1) that's known at compile "
          "time."),
```

Reply:
I think that this wording would be a bit confusing. How about:
>           "Minimum vector bitwidth to enable the flattening transformation. "
>           "For scalable vectors this is the base size, i.e. the size "
>           "corresponding to vscale=1."),

?

Reply:
Sure, or simply `"For scalable vectors this is the size corresponding to vscale=1."`. To me the important thing is mentioning vscale=1 as I think base size is a bit ambiguous.